voltdbclient/url_test.go

```diff
@@ -0,0 +1,50 @@
+package voltdbclient
+
+import (
+	"testing"
+)
+
+func TestParseURL(t *testing.T) {
+	sample := []struct {
+		conn                       string
+		pass                       bool
+		scheme, user, passwd, host string
+	}{
+		{"https://localhost:21212", true, "https", "", "", "localhost:21212"},
+		{"http://localhost:21212", true, "http", "", "", "localhost:21212"},
+		{"localhost:21212", true, "localhost", "", "", "localhost:21212"},
+		{"127.0.0.1:21212", false, "voltdb", "", "", "localhost:21212"},
```

Comment:
It is not a valid url, go won't parse it please see this playground  https://play.golang.org/p/S9avu_fGHy , run it and observe the output